geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -190,20 +191,48 @@ object Transformers extends EnhancedTokenParsers with LazyLogging {
     override def dependenciesOf(fieldNameMap: Map[String, Field]): Seq[String] = e.dependenciesOf(fieldNameMap)
   }
   case class Cast2Int(e: Expr) extends CastExpr {
-    override def eval(args: Array[Any])(implicit ctx: EvaluationContext): Any =
-      e.eval(args).asInstanceOf[String].toInt
+    override def eval(args: Array[Any])(implicit ctx: EvaluationContext): Int =
+      e.eval(args) match {
+        case null      => null.asInstanceOf[Int]
```

Comment:
this changes the behavior a bit - not sure if that is good or bad... previously a null string would fail and throw an NPE, now it will silently convert to 0:
```
scala> null.asInstanceOf[Int]
res0: Int = 0
```
@jahhulbert-ccri @jnh5y thoughts?